pom.xml

```diff
@@ -80,7 +80,7 @@
 		<dependency>
 			<groupId>org.isf</groupId>
 			<artifactId>OH-core</artifactId>
-			<version>1.9</version>
+			<version>2.0-SNAPSHOT</version>
```

Comment:
OH-core should remain 1.9 for compliance with other components

Reply:
Ummm...Are you sure about this choice? 
How can we check which core version is delivered with a specific OS version?

Reply:
Not sure I understood the question, the OH core will be the same for all OS versions and all components until we don't abandon the OH gui component, do you agree?

Reply:
Sorry, typo in my comment, i meant: 

> How can we check which core version is delivered with a specific O**H** version

Each version of OH could use a different version of OH-core.
For exemple:

OH 1.9.4 -> OH-core 1.9
OH 1.9.10 -> OH-core 2.0
OH 1.9.11 -> OH-core 2.0
OH 1.9.15 -> OH-core 2.1
OH-api 2.0 -> OH-core 2.0

Reply:
Yes, but at the moment we don't have yet a system for that and I think we should puslish everything on Maven Central one day. Meanwhile, one must have the source code of each component in order to compile, and we never had OH-core 2.0 up to now, nor a set a rules for the individual versions' increment.

The only rule we had up to now is on the overall application version, that is:
Version 1.X.Y where Y increments on small changes not involving the DB, X increments for changes involving the DB.

But I'd like to discuss it further in the next call, that we are about to schedule.